src/main/java/org/terasology/web/resources/systemStatus/SystemResource.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.web.resources.systemStatus;
+
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.web.resources.base.AbstractSimpleResource;
+import org.terasology.web.resources.base.ClientSecurityRequirements;
+import org.terasology.web.resources.base.ResourceAccessException;
+import org.terasology.web.resources.base.ResourceMethod;
+import org.terasology.web.resources.base.ResourcePath;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import static org.terasology.web.resources.base.ResourceMethodFactory.createParameterlessMethod;
+
+public class SystemResource extends AbstractSimpleResource {
+
+    private final ScheduledExecutorService refreshSystemInfoService = Executors.newScheduledThreadPool(1);
+
+    public void startSystemInfoRefreshService() {
+        refreshSystemInfoService.scheduleAtFixedRate(this::refreshSystemInfoForClient, 1000, 1000, TimeUnit.MILLISECONDS);
+    }
+
+    @ReceiveEvent
```

Comment:
Just one more note: the `@ReceiveEvent` annotation [is defined in the engine](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/entitySystem/event/ReceiveEvent.java), and is used to mark method inside `ComponentSystem` classes which can be used to handle events.
If you notice, in `OnlinePlayersResource`, `onConnected` and `onDisconnected` are never called directly (except in unit tests) - it's the engine which discovers them through reflection because the class implements the `ComponentSystem` interface and is annotated with `@RegisterSystem`, and calls them when appropriate. If you are interested in the topic, you may find [this page in the Terasology's wiki](https://github.com/MovingBlocks/Terasology/wiki/Events-and-Systems) useful.

It is not useful outside of `ComponentSystem`s.